src/components/index.tsx

```diff
@@ -1,21 +1,49 @@
 import Artwork from "./artwork"
 import ArtworkFilter from "./artwork_filter"
-import Buttons from "./buttons/index"
-import Modal from "./modal/modal"
-
 import ArtworkGrid from "./artwork_grid"
 import BorderedPulldown from "./bordered_pulldown"
+import Buttons from "./buttons/index"
 import Checkbox from "./checkbox"
 import Gene from "./gene/index"
 import Grid from "./grid"
 import Icon from "./icon"
 import Input from "./input"
+import Modal from "./modal/modal"
 import Nav from "./nav"
 import Text from "./text"
 import TextArea from "./text_area"
 import TextLink from "./text_link"
 import Title from "./title"
 
+import React from "react"
+import ReactDOM from "react-dom"
+import * as Relay from "react-relay"
+
+import { artsyNetworkLayer } from "../relay/config"
+import * as Artsy from "./artsy"
+
+interface InitOptions {
+  user: User,
+  component: any,
+  domID: string,
+  queryConfig: any,
+}
+
+export function init(options: InitOptions) {
```

Comment:
This function is what we will use to initialize reaction-force components on the client-side.